### PR TITLE
Add OTV back

### PR DESCRIPTION
**Pull request by link8k**
https://streamtest.in/logs/https-svs-itworkscdn-net-otvlebanonlive-otv-smil-playlist-m3u8-fd402263-ff1d-49d9-ab0f-f66bc25c2772

### CODE REVIEW

File: channels/lb.m3u [hunk 1]
```diff
@@ -31,6 +31,8 @@ https://svs.itworkscdn.net/nour8satlive/livestream/playlist.m3u8
 https://svs.itworkscdn.net/nour9satlive/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/QrwZu4W.png" group-title="Religious",NourSAT (576p)
 https://svs.itworkscdn.net/nour4satlive/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="OTV Lebanon ARB" tvg-name="OTV Lebanon ARB" tvg-language="Arabic" tvg-logo="http://bookback.co/wp-content/uploads/2018/12/OTV-1.jpg" group-title="General",OTV (1080p)
+https://svs.itworkscdn.net/otvlebanonlive/otv.smil/playlist.m3u8
```
Comment (freearhey) [inline channels/lb.m3u:35]: The link no longer works
Captura de pantalla 2021-01-28 a las 3 45 12

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/lb.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/lb.m3u:35]: maybe try : https://svs.itworkscdn.net/otvlebanonlive/livestream/playlist.m3u8

Comment (link8k) [reply channels/lb.m3u:35]: Thanks for checking but it's still working for me https://streamtest.in/logs/https-svs-itworkscdn-net-otvlebanonlive-otv-smil-playlist-m3u8-90cd8bfb-768a-4b7a-8278-3788feba1819

Maybe it was down before?

Comment (LaneSh4d0w) [reply channels/lb.m3u:35]: Or maybe it isn't 24/7

Comment (link8k) [reply channels/lb.m3u:35]: Not sure. I think it is 24/7

Comment (freearhey) [reply channels/lb.m3u:35]: Now it works for me, too.

**Review by freearhey — APPROVED** [review]

**Pull request merged**